### PR TITLE
release upgrade howto

### PR DESCRIPTION
**Pull request by razvan-moj**
**Why**
Steps as identified during the 1.13 upg

### CODE REVIEW

File: runbooks/source/upgrade-cluster.html.md.erb [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+kops export kubecfg XXXXXXX.cloud-platform.service.justice.gov.uk
+```
+
+Change the Kubernetes release version, ensuring it is supported by Kops - verify on the [kops releases](https://github.com/kubernetes/kops/releases) page.
+
+If there is a newer AMI available in https://github.com/kubernetes/kops/blob/master/channels/stable, `kops edit instance-group masters`
+
+`kops update cluster (--yes)` will update k8s components inc calico
+
+`kops rolling update cluster -- instance-group-roles=Master` will update only th emaster nodes, no performance impact expected at this point
```
Comment (jasonBirchall) [inline runbooks/source/upgrade-cluster.html.md.erb]: the > th

File: runbooks/source/upgrade-cluster.html.md.erb [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+
+`kops create instance group nodes-<new_group_name>` to double the size of the cluster
+
+ - copy values from instance group nodes
+
+`kops update cluster (--yes)` will create the new nodes
+
+`kubectl cordon nodes` in the old instance group, this will ensure any new resources will go to the new ig
```
Comment (jasonBirchall) [inline runbooks/source/upgrade-cluster.html.md.erb]: kubectl cordon nodes <node_name>

Comment (jasonBirchall) [reply runbooks/source/upgrade-cluster.html.md.erb]: and you can only do it one node at a time.

File: runbooks/source/upgrade-cluster.html.md.erb [hunk 3]
```diff
@@ -0,0 +1,123 @@
...
+
+Scale up the OPA deployment to ensure at least one pod runs on the new nodes
+
+Scale up ingress controllers by at least 3, to ensure each AZ on the new nodes gets an instance
+
+Identify pods with PVCs
```
Comment (jasonBirchall) [inline runbooks/source/upgrade-cluster.html.md.erb]: `kubectl get pvc -A`

File: runbooks/source/upgrade-cluster.html.md.erb [hunk 4]
```diff
@@ -0,0 +1,123 @@
...
+
+ - kill them
+
+ - check data persistence (eg Prometheus metrics history)
+
+`kubectl drain nodes` on the old ig
```
Comment (jasonBirchall) [inline runbooks/source/upgrade-cluster.html.md.erb]: same as above `kubectl drain nodes <node_name>`

Comment (jasonBirchall) [reply runbooks/source/upgrade-cluster.html.md.erb]: and one at a time.

File: runbooks/source/upgrade-cluster.html.md.erb (hunk 2 above)
Comment (razvan-moj) [reply runbooks/source/upgrade-cluster.html.md.erb]: `cordon` did work fine on all nodes, this is only for `drain` right?

Comment (jasonBirchall) [reply runbooks/source/upgrade-cluster.html.md.erb]: unfortunately cordon and drain :(

**Pull request merged**